js/blocks/PitchBlocks.js

```diff
@@ -603,6 +603,11 @@ function _playPitch(args, logo, turtle, blk) {
         logo.pushedNote[turtle] = true;
     } else if (logo.drumStyle[turtle].length > 0) {
         let drumname = last(logo.drumStyle[turtle]);
+        let transposition = 2 * delta;
```

Comment:
Not sure delta is defined either.
But I don't think we need it in this case. Just let transposition = 0 and add in the logo.transposition, it there is any.

Reply:
Done.